src/hooks/useConnectRelatedVenues.ts

```diff
@@ -235,11 +259,34 @@ export const useConnectRelatedVenues: ReactHook<
     [currentVenue, parentVenue, siblingVenues, subvenues]
   );
 
+  const isRelatedVenuesLoaded = useMemo(() => {
+    const loaders = [
+      isCurrentVenueLoaded,
+      isSubvenuesLoaded,
+      isSiblingVenuesLoaded,
+    ];
+
+    if (parentId) {
+      loaders.push(isParentVenueLoaded);
+    }
+
+    return loaders.reduce(
+      (finalIsLoaded, isLoaded) => finalIsLoaded && isLoaded
+    );
+  }, [
+    parentId,
+    isCurrentVenueLoaded,
+    isParentVenueLoaded,
+    isSubvenuesLoaded,
+    isSiblingVenuesLoaded,
+  ]);
+
   return useMemo(
     () => ({
       parentVenue,
       currentVenue,
       relatedVenues,
+      isRelatedVenuesLoaded,
```

Comment:
Yes, I do agree with you, but I think we should do it when there is an explicit need for this. For now, we could just add a ticket to the backlog

Reply:
I don't think it's worth adding this as a backlog ticket. For the sake of adding a couple of extra lines to the return value I would just do it now. It may be that we don't need them, but that isn't going to hurt anything/take us any real extra time. But it means they are there if/when they are useful to us. Exposing a 'powerful API' and such.